website/docs/reference/appsmith-framework/widget-actions/README.md

```diff
@@ -12,7 +12,7 @@ Browse this section to learn about the different actions you can trigger on Apps
 * [Store Value](/reference/appsmith-framework/widget-actions/store-value)
 * [Navigate To](/reference/appsmith-framework/widget-actions/navigate-to)
 * [Show Alert](/reference/appsmith-framework/widget-actions/show-alert)
-* [Open Modal](/reference/appsmith-framework/widget-actions/show-modal)
+* [Show Modal](/reference/appsmith-framework/widget-actions/show-modal)
```

Comment:
can you change all of this relevant function signature? same for everything on the other page heading except 

Intervals (Time Events)
Cross-origin Communication

so that the sidebar looks a bit consistent